packages/calcite-components/src/components/list/list.tsx

```diff
@@ -239,6 +241,9 @@ export class List
    */
   messages = useT9n<typeof T9nStrings>({ blocking: true });
 
+  /** Specifies the nesting behavior. */
+  @property({ reflect: true }) mode: ListMode = "flat";
```

Comment:
Should this be `displayMode` to match some other component nomenclature? This doesn't affect any selection or other functionality right, just visual?

Reply:
I'll defer to @jcfranco

Reply:
It does affect some functionality. `nested` must be used to add the expand button and indent children. That said, I could go either way here and also defer to @jcfranco.

Reply:
Good point. Let's go with `displayMode` for consistency. The only other `mode` prop I found was `input-time-zone`'s, which affects both the API and presentation.